bin/evaluate-measurements.js

```diff
@@ -97,27 +97,23 @@ async function processRound (roundIndex, measurements, resultCounts) {
     prepareProviderRetrievalResultStats: async () => {}
   })
 
-  for (const m of round.measurements) {
-    // FIXME: we should include non-majority measurements too
-    // See https://github.com/filecoin-station/spark-evaluate/pull/396
-    if (m.taskingEvaluation !== 'OK' && m.consensusEvaluation === 'MAJORITY_RESULT') continue
-    resultCounts.total++
-    resultCounts[m.retrievalResult] = (resultCounts[m.retrievalResult] ?? 0) + 1
+  if (!keepRejected) {
+    round.measurements = round.measurements.filter(m => m.taskingEvaluation === 'OK')
   }
 
-  if (!keepRejected) {
-    round.measurements = round.measurements
-      // Keep accepted measurements only
-      // FIXME: we should include non-majority measurements too
-      // See https://github.com/filecoin-station/spark-evaluate/pull/396
-      .filter(m => m.taskingEvaluation === 'OK' && m.consensusEvaluation === 'MAJORITY_RESULT')
-      // Remove the taskingEvaluation and consensusEvaluation fields as all accepted measurements have the same value
-      .map(m => ({ ...m, taskingEvaluation: undefined, majorityEvaluation: undefined }))
+  for (const m of round.measurements) {
+    resultCounts.total++
+    const status = m.taskingEvaluation !== 'OK'
```

Comment:
Maybe it's just me but this double ternary operator makes this really hard to read. In case we leave this loop logic do you think plain `if` statements would make this a bit more readable?

Reply:
cf85474